.github/workflows/build.yml

```diff
@@ -27,9 +27,6 @@ jobs:
       - name: Grant permissions for default user
         run: kubectl apply -f ./k8s/rbac.yaml
 
-      - name: Docker Build
```

Comment:
We will need this as `K8sInterface` and `K8sWorkerContoller` unit tests are spawning workers